.github/labeler.yml

```diff
@@ -16,25 +16,47 @@
 #
 
 inlong-agent:
-  - inlong-agent/**/*
+  - 'inlong-agent/**/*'
 
 inlong-dataproxy:
-  - inlong-dataproxy/**/*
+  - 'inlong-dataproxy/**/*'
 
 inlong-manager:
-  - inlong-manager/**/*
+  - 'inlong-manager/**/*'
 
 inlong-sort:
-  - inlong-sort/**/*
+  - 'inlong-sort/**/*'
+
+inlong-sort-standalone:
+  - 'inlong-sort-standalone/**/*'
 
 inlong-tubemq:
-  - inlong-tubemq/**/*
+  - 'inlong-tubemq/**/*'
 
 inlong-dashboard:
-  - inlong-dashboard/**/*
+  - 'inlong-dashboard/**/*'
 
 inlong-audit:
-  - inlong-audit/**/*
+  - 'inlong-audit/**/*'
 
 inlong-sdk:
-  - inlong-sdk/**/*
+  - 'inlong-sdk/**/*'
+
+ci:
+  - '.github/workflows/**/*'
+  - '.github/actions/**/*'
+
+docker:
+  - '**/Dockerfile'
+  - '**/*-docker.sh'
+  - 'inlong-agent/agent-docker/**/*'
+  - 'inlong-audit/audit-docker/**/*'
+  - 'inlong-dataproxy/dataproxy-docker/**/*'
+  - 'inlong-manager/manager-docker/**/*'
+  - 'inlong-tubemq/tubemq-docker/**/*'
+  - '!inlong-tubemq/tubemq-docker/tubemq-k8s/**/*'
+  - 'docker/docker-compose/**/*'
+
+k8s:
+  - 'docker/kubernetes/**/*'
+  - 'inlong-tubemq/tubemq-docker/tubemq-k8s/**/*'
```

Comment:
@dockerzhang Are these two labels OK? And are some else labels needed?

Reply:
it's ok.